internal/labels/labels.go

```diff
@@ -0,0 +1,12 @@
+package labels
+
+const (
+	OwnerKindKey = "olm.operatorframework.io/owner-kind"
+	OwnerNameKey = "olm.operatorframework.io/owner-name"
+
+	// Helm Secret annotations use the regex `(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])?`
+	// to validate labels. Open to any suggestions, keeping this for now.
```

Comment:
Could someone please provide more context for this?

Reply:
There is a limitation on helm annotations. We'll add additional context.
OWNER: @varshaprasad96

Reply:
Fixed in https://github.com/operator-framework/operator-controller/pull/887. 

Tldr: This comment is to highlight that the current label key is not in the usual format with `_` and `/` which we follow for Owner annotations. Shouldn't be an issue at all, since it's just a label format.